app/params/config.go

```diff
@@ -17,6 +17,15 @@ const (
 
 	// Bech32PrefixAccAddr defines the Bech32 prefix of an account's address.
 	Bech32PrefixAccAddr = "bbn"
+
+	// BaseCosmosDenom is the base denomination for 6 decimals.
+	BaseCosmosDenom = "ubbn"
+	// BaseEVMDenom is the base denomination for 18 decimals.
+	BaseEVMDenom = "abbn"
+	// DisplayDenom is the display denom of the Cosmos EVM
+	DisplayDenom = "bbn"
```

Comment:
is this different from `HumanCoinUnit`? or we can use that instead of adding a new const?